components/events/SpeechCard.vue

```diff
@@ -1,25 +1,46 @@
 <template>
-    <locale-link v-show="shouldShow" :to="to" class="speechCard">
-        <div class="speechCard__speakerThumbnails">
-            <div
-                v-for="(speaker, i) in speakers"
-                :key="`speech_card_speaker_thumbnail_${i}`"
-                class="speechCard__speakerThumbnail"
-            >
-                <img :src="speaker.thumbnail_url" :alt="speaker.name" />
+    <locale-link v-show="shouldShow" :to="to" class="speechCard flex flex-col">
+        <div>
+            <figure
+                class="speechCard__levelTriangle"
+                :style="getLevelTriangleStyle(level)"
+            ></figure>
+            <div class="speechCard__level">
+                {{ $t(`levels.${level}`) }}
+            </div>
+            <div class="text-right">
+                <div class="speechCard__date">
+                    {{ getDateTag(beginTime) + ' ' + getTime(beginTime) }}
+                </div>
+                <div class="speechCard__location">
+                    {{ locationMapping[location] }}
+                </div>
             </div>
         </div>
-        <div class="speechCard__sub">
-            <p class="speechCard__speakerNames">
-                {{ speakers.map((speaker) => speaker.name).join(' • ') }}
-            </p>
-            <div class="speechCard__icon">
-                <img :src="icon.level[level]" />
-                <img :src="icon.lang[lang]" />
+        <div class="speechCard__title speechCard__title-overflow">
+            {{ title }}
+        </div>
+        <div class="flex justify-between items-center">
+            <div class="speechCard__speakerThumbnails">
+                <div
+                    v-for="(speaker, i) in speakers"
+                    :key="`speech_card_speaker_thumbnail_${i}`"
+                    class="speechCard__speakerThumbnail"
+                >
+                    <img :src="speaker.thumbnail_url" :alt="speaker.name" />
+                </div>
             </div>
+            <div
+                class="speechCard__lang"
+                :style="getMaskImgStyle(icon.lang[lang])"
+            ></div>
+        </div>
+        <div class="speechCard__speakerNames flex-1">
+            {{ speakers.map((speaker) => speaker.name).join('、') }}
```

Comment:
I think using Eng. punctuation here is more reasonable

```suggestion
            {{ speakers.map((speaker) => speaker.name).join(', ') }}
```

Reply:
[b4c47bb](https://github.com/pycontw/pycontw-frontend/pull/259/commits/b4c47bb54b8e2903ce43676399d7e9079cab2a57)

Reply:
42b8b61